src/tests/validity/import_with_trailing_comma.ab

```diff
@@ -0,0 +1,13 @@
+import {
+    replace,
+    replace_regex,
+} from "std/text"
+import { sum, abs } from "std/math"
+import {} from "std/env"
```

Comment:
Not a big deal, but it may look slightly nicer with an extra space between the braces for consistency with the other `import` statements:
```
import { } from "std/env"
```